### DISCUSSION TITLE
"Unknown token" in log when running on Boxcar DIA files

### DISCUSSION THREAD

**Discussion by adi772** [vdemichev/DiaNN#482] [Q&A]
Hi Dr. Demichev, thank you for again for this tool and for all your help.

When I am running DIA-NN files, I notice that the log is dominated by "Unknown token: x - y " where X is the start and Y is the end of a window. This doesn't appear in the final output log, so I have taken a screenshot while it is running.
![image](https://user-images.githubusercontent.com/110159670/187321000-a486565d-2801-4cb2-8848-318835b79e19.png)

I wonder if this causing any problems with the analysis. The output seems to be okay in terms of expected IDs and abundances of proteins. 

I have attached the full log here as well.
[acetonevdryreport.log.txt](https://github.com/vdemichev/DiaNN/files/9448574/acetonevdryreport.log.txt)

**Comment by vdemichev** [comment]
This is definitely not normal. Maybe something related to BoxCar - the mstoolkit library that DIA-NN uses to read .raw files complaining about this... Most likely no detrimental effects from this.

Best,
Vadim
